projects/ngrx.io/content/navigation.json

```diff
@@ -123,6 +123,10 @@
                 {
                   "title": "Downgrade for AngularJS",
                   "url": "guide/store/recipes/downgrade"
+                },
+                {
```

Comment:
This should be under `Recipes` at the top-level as the rules span across multiple packages.